.github/workflows/main.yml

```diff
@@ -28,9 +28,26 @@ jobs:
         with:
           framework: kubernetes
 
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - uses: Alfresco/alfresco-build-tools/.github/actions/validate-maven-versions@v1.22.0
```

Comment:
I wonder if this helm chart should be the one validating versions: I'm puzzled by the fact that requires making it a maven module

Reply:
It is going to be very cumbersome to validate versions via Helm chart. The Maven version validation already exists, so it only makes sense to reuse it.